Switch references from libpod.conf to containers.conf (#7009)
containers/podman: docs/source/markdown/podman.1.md

**--cgroup-manager**=*manager*

> CGroup manager to use for container cgroups. Supported values are cgroupfs or systemd. Default is systemd unless overridden in the containers.conf file.

Reviewer: ```suggestion
The cgroup manager to use for container cgroups. Supported values are cgroupfs or systemd. Default is systemd unless overridden in the containers.conf file.
```

## FILES

**containers.conf** (`/usr/share/containers/containers.conf`)

>     containers.conf is the configuration file for container engines.  Administrators can override the defaults file by creating `/etc/containers/containers.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/containers.conf` is created and replaces some fields in the system configuration file.

Reviewer: ```suggestion
    containers.conf is the configuration file for container engines, when run as root.  Administrators can override the defaults file by creating `/etc/containers/containers.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/containers.conf` is created and replaces some fields in the system configuration file.
```

Author: @TomSweeneyRedHat How about the new wording.

## FILES

**containers.conf** (`/usr/share/containers/containers.conf`)

    Podman has builtin defaults for command line options. These defualts can be overridden using the containers.conf configuration files.

> Distributions ship the `/usr/share/containers/containers.conf` file with their defaults. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the specify fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution files, which override the builtin defaults.

Reviewer: ```suggestion
Distributions ship the `/usr/share/containers/containers.conf` file with the default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify the defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the fields from these files, if they exist. Fields specified in the user's file override the administrator's file, which overrides the distribution's file, which override the built-in defaults.
```

Reviewer: tweaked slighly @rhatdan, but I like where you're going with it.

> Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the specify fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution's file, which override the built-in defaults.

Reviewer: ```suggestion
Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the specified fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution's file, which override the built-in defaults.
```

> Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the specify fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution's file, which override the built-in defaults.

Reviewer: specify to specified in the prior, doesn't show well in the diff

> Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the specify fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution's file, which override the built-in defaults.

Author: Fixed

> Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges its builtin defaults with the specified fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution's file, which override the built-in defaults.

Reviewer: sorry if I missed this this morning, missing a possesive adminstrator.
```suggestion
Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges its builtin defaults with the specified fields from these files, if they exist. Fields specified in the users file override the administrator's file, which overrides the distribution's file, which override the built-in defaults.
```

>     Podman uses builtin defaults if no containers.conf file is found.

Reviewer: space/tab issue on this line?

Author: Fixed
